dashboard/test/ui/test_status.js

```diff
@@ -171,7 +172,7 @@ function testFromS3Key(key) {
   // unlikely.
   var test = tests[browser][feature];
   while (!test && feature.indexOf('_') >= 0) {
-    feature = feature.replace('_', '/'); // Replaces first underscore only.
+    feature = feature.replace(/_/, '/'); // Replaces first underscore only.
```

Comment:
This file doesn't get preprocessed in any way, so it shouldn't be an ES6 or transpiling thing.  I just noticed while doing some debugging.

I like using regex everywhere I can anyway :trollface: